inc/job-listing.php

```diff
@@ -9,7 +9,7 @@
 	 **/
 
 	function sanitizeAddress($address) {
-		$allCaps = ["HM","HMP","YOI","HMP/YOI","HMYOI","HMPYOI","NPS","NMS","RCJ","OPG","JCP","LAA","HMCTS","HMPPS","CTS","LG"];
+		$allCaps = ["HM","HMP","YOI","HMP/YOI","HMYOI","HMPYOI","NPS","NMS","RCJ","OPG","JCP","LAA","HMCTS","HMPPS","CTS","LG,"];
```

Comment:
Did you mean to add the comment in the end here?

Reply:
Technically, shouldn't cause an issue, but just wondering as the other array in the file doesn't have it.

Reply:
As it is in the quotes it just is part of the string, so should be okay.  It seems to work as intended.